### PR TITLE
Fixed the link to the proper CC0 license page

### PR DESCRIPTION
**Pull request by Kezxo**
The original link was a page about the license, not the actual license.
Also the very first link to ``license.md`` leads to a file that's not there. I'm not sure what that file should contain, just letting you know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ All scripts, datasets, and documentation are [permissively and openly licensed](
 
 * All **media components** (text, figures, etc.) are licensed under the [Attribution International (CC BY) 4.0 License](https://creativecommons.org/licenses/by/4.0/)
 * All **code** is licensed under the permissive open source [MIT License](https://opensource.org/licenses/MIT)
-* All **datasets** are licensed under the [Creative Commons CC0 License](https://creativecommons.org/share-your-work/public-domain/cc0/)
+* All **datasets** are licensed under the [Creative Commons CC0 License](https://creativecommons.org/publicdomain/zero/1.0/)
```
Comment (4383) [inline README.md]: If you want to update the link then the link's text should be updated to `CC0 1.0` too:

```
* All **datasets** are licensed under the [Creative Commons CC0 1.0 License](https://creativecommons.org/publicdomain/zero/1.0/)
```

**Review by 4383 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Kezxo) [reply README.md]: Sure, that makes sense. I changed it, hope it was the most correct way...

**Comment by 4383** [comment]
Thanks, also it could be worth to [squash your commits](https://davidwalsh.name/squash-commits-git) to keep a proper history:
```
$ git rebase -i HEAD~2
# now you are in interactive mode you should fixup the latest commit
# Normally it should look like to something like:
pick cf2aa30 Fixed the link to the proper CC0 license page
fixup 3a50bac Changed the license link text.
```
And then you just have to push your changes with `--force` (`git push --force origin patch-1`)
Thanks for you efforts on it :)

**Comment by jchodera** [comment]
Thanks! Apologies for letting this PR go for so long!

**Pull request merged**